src/coreclr/tools/Common/Compiler/DisplayNameHelpers.cs

```diff
@@ -116,7 +117,58 @@ public static string GetDisplayName(this EventPseudoDesc @event)
 
         public static string GetDisplayName(this TypeDesc type)
         {
-            return Formatter.Instance.FormatName(type, FormatOptions.NamespaceQualify);
+            var sb = new StringBuilder();
+            if (type == null)
```

Comment:
Asking for a display name of null sounds like a bug. We're going to print a useless message to the user.